src/main/features/response/form/models/statement-of-means/bankAccounts.ts

```diff
@@ -11,15 +10,13 @@ export class ValidationErrors {
 
 export class BankAccounts extends MultiRowForm<BankAccountRow> {
 
-  @AtLeastOnePopulatedRow({ message: ValidationErrors.AT_LEAST_ONE_ROW_REQUIRED })
   rows: BankAccountRow[]
 
   static fromObject (value?: any): BankAccounts {
     if (!value) {
       return value
     }
-
-    return new BankAccounts(value.rows ? value.rows.map(BankAccountRow.fromObject) : [])
+    return new BankAccounts(value.rows ? value.rows.map(BankAccountRow.fromObject) : undefined)
```

Comment:
do you have to update test to return undefined instead of [] ??

Reply:
No, actually we can put it back to [] now
```suggestion
    return new BankAccounts(value.rows ? value.rows.map(BankAccountRow.fromObject) : [])
```